Vagrantfile

```diff
@@ -9,11 +9,22 @@ if Gem::Version.new(Vagrant::VERSION) < Gem::Version.new("1.5.0")
 	exit 1
 end
 
+# Path to the root of the Chassis directory.
+chassis_dir = File.dirname(__FILE__)
+
+# Load Chassis config from the Chassis root by default.
+chassis_config_dir = chassis_dir
+
+# Load Chassis config overrides from a custom directory, if configured.
+unless ENV['CHASSIS_CWD'].nil?
+	chassis_config_dir = ENV['CHASSIS_CWD']
+end
+
 # Check that submodules have been loaded
-if not File.exist?(File.join(File.dirname(__FILE__), "puppet", "modules", "apt", ".git"))
+if not File.exist?(File.join(chassis_dir, "puppet", "modules", "apt", ".git"))
```

Comment:
`File.dirname(__FILE__)` was repeated in several places so we introduce `chassis_dir`.